src/controllers/profile.ts

```diff
@@ -1,25 +1,72 @@
+import dotenv from "dotenv";
 import { Request, Response } from "express";
 import { jwtDecode } from "jwt-decode";
 import Profile from "../models/profileModel";
 import { dateToZodiac } from "../services/dateToZodiac";
 import { setLocation } from "../services/location";
+import { ok } from "assert";
+import fs from "fs";
+
+dotenv.config();
+
+const cloudinary = require('cloudinary').v2;
```

Comment:
we already have cloudinary service and middleware written, please either edit the existing code or change yours to organize it properly and remove the old code